src/Popover/index.js

```diff
@@ -174,6 +185,9 @@ const Popover = ({
 // =======================================================
 
 Popover.propTypes = {
+   anchorElement: PropTypes.object,
```

Comment:
Thanks for adding this. Actually it should be a `node` though since it's referring to a DOM element right?